### ISSUE TITLE
Any Plans for Linux Support?

### ISSUE THREAD

**Issue by ash-hashtag** [T-Troll/alienfx-tools#255]
**Is your feature request related to a problem? Please describe.**
No

**Describe the solution you'd like**
Make this tool available for linux

**Describe alternatives you've considered**
not use linux

**System (please complete the following information):**
 - Model: Dell g15 5505
 - OS: Windows 11 Stable, but may go to some linux distro

**Additional context**
Windows is acting pretty strange lately, it became so buggy and slow, and I tried to clean something and I ignored that slow behaviour for a very long time, but now it is totally pissing me off, wsl stopped working, microsoft store stopped working(won't even let me uninstall apps), and now I just tried to turn off and on wsl feature for one last hope, and guess what? that windows search doesn't work anymore, doesn't even pop up, till now it atleast worked half the time. no restarts, troubleshooters, sfc fixed anything

Ofcourse this has nothing to do with these tools. So I can either reinstall windows, recovery options are also not working, and I don't remember having restore points either, or completely ditch windows and go full linux.

TLDR;
only thing that is keeping me is this laptop's fan control system, I've used linux in dual boot before, and I used to set that stupid 3 dot curve in awcc before and use linux, and it worked most of the time. 

Thank you, if you have any plans I'll look forward, or I have enough knowledge to try port it, but I wouldn't call myself professional, I don't want to take any risks, so atleast help the bits out.

**Comment by T-Troll** [comment]
I only use Linux as WSL, so i can't do it.

Clone repo and port the app yourself - it's quite easy for CLI.
